src/game.cpp

```diff
@@ -12051,13 +12051,17 @@ void game::autosave()
 
 void game::start_calendar()
 {
-    // Configured starting date overridden by scenario, calendar::start is left as Spring 1
```

Comment:
why remove the comment?

Reply:
It got accidentally removed when I collapsed the two branches. I readded it.